### PR TITLE
feat: :sparkles: Update scopeTag

### PR DESCRIPTION
**Pull request by MagnusGjosund**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

**Comment by MagnusGjosund** [comment]
Added the new columns to TagTable. 

Missing changes on HtTable.

File: src/apps/ReleaseControl/components/Form/Inputs/Scope/HtCableTable.tsx [hunk 1]
```diff
@@ -63,42 +71,94 @@ const columns: Column<RcScopeHtTag>[] = [
...
+        },
+        width: 70,
+    },
+    {
+        id: 'signedDate',
+        Header: 'Signed date',
+        accessor: (item) => item.signedDate,
+        width: 100,
+        Cell: (cell) => {
+            return (
```
Comment (Gustav-Eikaas) [inline src/apps/ReleaseControl/components/Form/Inputs/Scope/HtCableTable.tsx:104]: You could consider rendering <></> if signed date is null. Right now you are returning an empty monospace div

File: src/apps/ReleaseControl/components/Form/Inputs/Scope/HtCableTable.tsx [hunk 2]
```diff
@@ -63,42 +71,94 @@ const columns: Column<RcScopeHtTag>[] = [
...
                 return (
                     <StyledCenterCheckIcon>
-                        {Number(cell.value.content.installedCableLength)}
                         <Icon
                             color={tokens.colors.interactive.success__text.hex}
                             name="check_circle_outlined"
-                            title="This cable is installed."
+                            title="Cable installed"
                         />
+                        {Number(cell.value.content.installedCableLength)}
```
Comment (Gustav-Eikaas) [inline src/apps/ReleaseControl/components/Form/Inputs/Scope/HtCableTable.tsx:148]: Why is installed cable length of type string??

**Review by Gustav-Eikaas — APPROVED** [review]
Looks good! I think we should look into why estimatedCableLength is of type string and discuss a solution.

**Pull request merged**